### PR TITLE
Design optimizations in cart and checkout

### PR DESCRIPTION
**Pull request by paeddl**
| Questions | Answers |
| --- | --- |
| Branch? | develop |
| Description? | CSS adaption |
| Type? | bug fix |
| Category? | FO |
| BC breaks? | no |
| Deprecations? | no |
| Fixed ticket? | http://forge.prestashop.com/browse/BOOM-1085 |

### CODE REVIEW

File: themes/classic/templates/checkout/_partials/order-confirmation-table.tpl [hunk 1]
```diff
@@ -2,6 +2,7 @@
 <div id="order-items" class="col-md-8">
   <h3 class="card-title h3">{l s='Order items' d='Shop.Theme.Checkout'}</h3>
 {/block}
+<div class="order-confirmation-table">
```
Comment (maximebiloe) [inline themes/classic/templates/checkout/_partials/order-confirmation-table.tpl]: Bad indent. Could you fix it, please?

Comment (paeddl) [inline themes/classic/templates/checkout/_partials/order-confirmation-table.tpl]: @maximebiloe  done:
https://github.com/PrestaShop/PrestaShop/pull/5882/commits/222b36354a60ef403c38d30b9241932681d044bc

**Comment by maximebiloe** [comment]
Thank you!

**Comment by vincentbz** [comment]
Hi @paeddl ,

Thanks for that PR. Could you make some small fixes

Align the "per unit" price with the product price, there is a difference there
image

In the BOOM-1096, the borders around shipping method and items are missing
Here is what I have
image

Instead of 
image

thanks :)

**Comment by paeddl** [comment]
Hi @vincentbz 
did you recompile the css?

**Comment by vincentbz** [comment]
hmm no, thanks for the answer :)

**Pull request merged**